index.js

```diff
@@ -37,8 +38,36 @@ module.exports = (filePath, options) => {
 	if (optionsObj.verbose) {
 		console.log(chalk.gray(cmd));
 	}
-
+	var diff = [];
 	const resultBuffer = execSync(cmd);
+	const other_cmd = "grep -inRw -E 'TODO' . --exclude-dir={node_modules,test}"
+	const foo = function(cb){
+		exec(other_cmd, (err, stdout, stderr) => {
```

Comment:
Is there a particular reason we need the asynchronous exec here? If so, can you promisify it (I don't like callbacks):

```
const execute = (cmd) => new Promise((resolve, reject) => exec(cmd, (err, stdout, stderr) => err ? reject(err) : resolve({ stdout, stderr })));
```